helm/kyverno-policies-connectivity/templates/proxy/PodController.yaml

```diff
@@ -43,6 +43,15 @@ spec:
             - {{ . | quote }}
           {{- end }}
 {{- end }}
+        - resources:
+            kinds:
+            - Pod
+          selector:
+            matchExpressions:
+              - key: cnpg.io/cluster
+                operator: In
+                values:
```

Comment:
Maybe we should use the exists operator instead of in so we target all cnpg clusters?

Reply:
Good point :)
Done :heavy_check_mark: